### PR TITLE
Local Storage Fix

### LINKED ISSUE #234: local storage configuration
how would i configure the following in a server profile template:
Integrated storage controller
Mode Managed by OneView
Write cache Managed manually

Name | Type | RAID Level | Number of Drives | Size GB | Drive Technology | Boot | Accelerator
-- | -- | -- | -- | -- | -- | -- | --
OS | Logical drive | RAID 1 | 2 | n/a | not specified | Yes | Managed manually

### PR DESCRIPTION
**Pull request by AsisBagga**
### Description
Local Storage Fix in SPT

### Issues Resolved
#234 

### Check List
- [ ] New functionality includes testing.
  - [ ]  All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: oneview/data_source_server_profile_template.go [hunk 1]
```diff
@@ -169,6 +307,59 @@ func dataSourceServerProfileTemplateRead(d *schema.ResourceData, meta interface{
...
+	}
+	local_storage = map[string]interface{}{
+		"manage_local_storage": spt.LocalStorage.ManageLocalStorage,
+		"initialize":           spt.LocalStorage.Initialize,
+		"controller":           controllers,
+		"sas_logical_jbod":     sasLogDrives,
+	}
+
+	file, _ := json.MarshalIndent(local_storage, "", " ")
+	_ = ioutil.WriteFile("test.json", file, 0644)
```
Comment (nabhajit-ray) [inline oneview/data_source_server_profile_template.go:361]: remove this

Comment (AsisBagga) [reply oneview/data_source_server_profile_template.go]: done

**Comment by rismoney** [comment]
btw - the work you are doing is awesome.  so excited to get this functionality!

**Review by nabhajit-ray — APPROVED** [review]

**Comment by AsisBagga** [comment]
> btw - the work you are doing is awesome. so excited to get this functionality!

Appreciate it Rich. Thank you.

**Pull request closed without merging**